src/docfx/build/moniker/MonikerProvider.cs

```diff
@@ -80,7 +81,7 @@ public MonikerList GetZoneLevelMonikers(ErrorBuilder errors, FilePath file, Sour
             // User should not define it in moniker zone
             if (configMonikerRange.Value is null && ValidateMoniker(file))
```

Comment:
consider rename it to "should validate moniker", do the same for the variable in line 115. Because `ValidateMoniker` is the action to actually validate the moniker.

```suggestion
            if (configMonikerRange.Value is null && ShouldValidateMoniker(file))
```